clientdb/order_test.go

```diff
@@ -192,6 +193,7 @@ func dummyOrder(amt btcutil.Amount, leaseDuration uint32) *order.Kit {
 		panic(fmt.Sprintf("could not create private key: %v", err))
 	}
 	kit := order.NewKitWithPreimage(testPreimage)
+	kit.AuctionType = order.BTCInboundLiquidity
```

Comment:
This is making the default one explicit. I am creating a new kit, so it will have the zero value,  which is `order.BTCInboundLiquidity`. Then for testing that serialization works I create an ask with the default `dummyKit` and then overwrite it with something that is not the zero value, only for that test.